tests/DqtApi.Tests/DqtApi.Tests.csproj

```
@@ -1,14 +1,18 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
 
     <IsPackable>false</IsPackable>
     <NoDefaultLaunchSettingsFile>true</NoDefaultLaunchSettingsFile>
+    <UserSecretsId>7f5e1670-9172-4191-ae15-62f237715d18</UserSecretsId>
```

Might be better to have a 'pretty' string here so it's easier to manipulate secrets from the command line.
```suggestion
    <UserSecretsId>DqtApiTests</UserSecretsId>
```